index.html

```diff
@@ -9,6 +9,6 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <img class="title" src="front-end-tools.png" alt="image-title"></img>
```

Comment:
By convention we name files using camelCase. BTW, as this image will be reused on `Back-End Tools` section title too, I suggest you to name it `sectionTitleImage.png` or something like that :smile: